components/List/tests/snapshot/MDCSelfSizingStereoCellSnapshotTests.m

```diff
@@ -42,11 +70,41 @@ - (void)tearDown {
   [super tearDown];
 }
 
+- (UIWindow *)generateWindowWithView:(UIView *)view
+                 contentSizeCategory:(UIContentSizeCategory)sizeCategory
+                              insets:(UIEdgeInsets)insets {
+  MDCSelfSizingStereoCellSnapshotTestsContentSizeCategoryOverrideWindow *backgroundWindow =
+      [[MDCSelfSizingStereoCellSnapshotTestsContentSizeCategoryOverrideWindow alloc]
+          initWithFrame:CGRectMake(0, 0, CGRectGetWidth(view.bounds) + insets.left + insets.right,
+                                   CGRectGetHeight(view.bounds) + insets.top + insets.bottom)];
+  backgroundWindow.contentSizeCategoryOverride = sizeCategory;
+  backgroundWindow.backgroundColor = [UIColor colorWithWhite:(CGFloat)0.8 alpha:1];
+  [backgroundWindow addSubview:view];
+  [backgroundWindow makeKeyAndVisible];
```

Comment:
This usually crashes in a headless environment - is it passing for you locally using Bazel?

Reply:
I can change it to `.hidden = NO` if you feel more comfortable with that.

Reply:
I don't mind it being not hidden, I'm surprised that we need to make it the `keyWindow` for the test to pass.  Feel free to keep it as-is, but a comment here or inline comment about its necessity would be welcome.

Reply:
Addressed in 7ec0b163eadea7f4f28a86396413e7b4f094b96d.